sessionctx/variable/session.go

```diff
@@ -1181,6 +1182,8 @@ const (
 	SlowLogCopWaitMax = "Cop_wait_max"
 	// SlowLogCopWaitAddr is the address of TiKV where the cop-task which cost wait process time run.
 	SlowLogCopWaitAddr = "Cop_wait_addr"
+	// SlowLogBackoffPrefix contains backoff information.
+	SlowLogBackoffPrefix = "Backoff_"
```

Comment:
how about add Cop_ prefix to this~? or maybe help let 2pc's backoff time can be get in slow log too :smile:

Reply:
Updated, it is Cop_backoff_ now.